crates/iota-data-ingestion/src/workers/kv_store.rs

```diff
@@ -118,9 +115,6 @@ impl KVStoreWorker {
         let mut backoff = ExponentialBackoff::default();
         let mut queue: VecDeque<Vec<_>> = items.chunks(25).map(|ck| ck.to_vec()).collect();
         while let Some(chunk) = queue.pop_front() {
-            if instant.elapsed() > TIMEOUT {
-                return Err(anyhow!("key value worker timed out"));
-            }
```

Comment:
It could be an option to keep the timeout, but reset it before the next iteration in this `while` loop. This way we still force a timeout for each chunk.

Reply:
scratch that. The request itself will either succeed or it will fail, and we do have the backoff for any unprocessed items.